conda_build/metadata.py

```diff
@@ -1789,6 +1818,27 @@ def get_used_loop_vars(self, force_top_level=False):
         return {var for var in self.get_used_vars(force_top_level=force_top_level)
                 if var in self.get_loop_vars()}
 
+    def get_rendered_outputs_section(self):
+        extract_pattern = r'(.*)package:'
+        template_string = '\n'.join((self.get_recipe_text(extract_pattern=extract_pattern,
+                                                          force_top_level=True),
+                                    # second item: the output text for this metadata
+                                    #    object (might be output)
+                                    self.extract_outputs_text())).rstrip()
+
+        outputs = (yaml.safe_load(self._get_contents(permit_undefined_jinja=False,
+                                template_string=template_string)) or {}).get('outputs', [])
+        self.parse_until_resolved()
+        return get_output_dicts_from_metadata(self, outputs=outputs)
+
+    def get_rendered_output(self, name):
```

Comment:
This is for obtaining the rendered, parsed, dictionary-object representation of an output.  It's not useful for saying what variables are used.  You need earlier, more raw versions of the metadata for that.  It is useful, however, for getting updated, re-rendered contents of outputs.